README.md

```diff
@@ -9,3 +9,5 @@ Security advisories for issues found by Asymmetric Research. Please also see our
 | [Evmos - Precompile State Commit Infinite Mint](./advisories/2024-08-01_evmos_precompile-state-commit-ininite-mint.md) | Due to incorrect state commits during precompile execution, it is possible to introduce discrepancies in the EVM state that could be abused to mint native tokens.| [Jason Matthyser](https://x.com/pleasew8t) |
 | [Circle - Noble CCTP Mint Bug](./advisories/2024-08-27_circle_noble-cctp-mint-bug.md) | A vulnerability that could have been exploited by circumventing the CCTP message sender verification process to potentially mint fake USDC tokens on Noble. | [Ruslan Habalov](https://x.com/evonide) |
 | [Ethereum - Ghost in the Block](./advisories/2024-09-19_ethereum-ghost-in-the-block.md) | Slight nuances in SSZ deserialization between the Prysm and Lighthouse clients could have allowed an attacker to degrade Ethereum consensus. | [Giuseppe Cocomazzi](https://www.sbudella.altervista.org) | 
+| [CometBFT - Missing BLS12381 Public Key Validation](./advisories/2024-09-25_cometbft-bls12381-key-validation.md) | Omission of the subgroup check for public keys allows for computations in small subgroups. | [Giuseppe Cocomazzi](https://www.sbudella.altervista.org) |
```

Comment:
I see you are using `BLS12381` here, and `BLS12-381` in other places. Are they two different things, or just using the names interchangeably? Could be made the same if they are the same.